src/messaging/ExchangeMgr.h

```diff
@@ -87,16 +87,17 @@ class DLL_EXPORT ExchangeManager : public SessionMessageDelegate
     /**
      *  Creates a new ExchangeContext with a given peer CHIP node specified by the peer node identifier.
      *
-     *  @param[in]    session    The identifier of the secure session (possibly
-     *                           the empty session for a non-secure exchange)
-     *                           for which the ExchangeContext is being set up.
+     *  @param[in]    session       The identifier of the secure session (possibly
+     *                              the empty session for a non-secure exchange)
+     *                              for which the ExchangeContext is being set up.
      *
-     *  @param[in]    delegate   A pointer to ExchangeDelegate.
+     *  @param[in]    delegate      A pointer to ExchangeDelegate.
+     *  @param[in]    isInitiator   Set to true if the exchange is created on the initiator.
```

Comment:
Maybe document that this should generally be true except for unit tests?

Or maybe we should have a unit-test-only thing for creating responder exchanges...